client/src/app/pages/applications/useDecoratedApplications.ts

```diff
@@ -60,6 +62,8 @@ export interface DecoratedApplication extends Application {
     identities?: Identity[];
     archetypes?: Archetype[];
   };
+
+  manifest?: Manifest;
```

Comment:
Since this is only useful to the detail drawer tab that would take at least 2 user clicks to get to, there is no need to fetch the manifest for all applications here.

Instead, have the tab contents component pull the application manifest directly and pass it along to the editor.